pkg/remotewrite/remotewrite.go

```diff
@@ -15,7 +16,15 @@ import (
 	prompb "go.buf.build/grpc/go/prometheus/prometheus"
 )
 
-var _ output.Output = new(Output)
+var (
+	_ output.Output = new(Output)
+
+	// staleNaN is the Prometheus special value for marking
+	// a time series as stale.
+	//
+	// https://pkg.go.dev/github.com/prometheus/prometheus/pkg/value#pkg-constants
+	staleNaN = math.Float64frombits(0x7ff0000000000002)
```

Comment:
Would it make sense to make this particular value a constant?

Reply:
I'm looking into this but I'm not sure that the alternative is really better because the const would be a uint64 and it would be accepted if assigned to a float64. So if we forget to convert with `math.Float64frombits(` then we introduce a bug because the compiler doesn't complain. We should only rely on unit tests. @oleiade WDYT?

Reply:
>Can't we just use the StaleNaN constant from the prometheus package directly?

@imiric It would require including the Prometheus project as a direct dependency, but it has a lot of dependencies.